app/views/projects/show.html.erb

```diff
@@ -31,11 +31,36 @@
 <%= link_to 'Edit', edit_project_path(@project), class: "btn btn-primary btn-md" unless current_user.user? %>
 <%= link_to 'Back', projects_path, class: "btn btn-info btn-md" %>
 
+<% if @project.attachments.any? %>
```

Comment:
why not `blank?`

Reply:
if i will use blank I have to use unless.

Reply:
But I should use blank bcz it is an ActiveSupport.